ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+> Please label the issue either bug or proposal according to its purpose.
+
+### Version
+> Please state the version of the UI.
+
+### Description
+> Please describe the bug or your proposed change to the software as detailed as possible. If screenshots can be used to describe the bug/proposal, we are thankful if you provide it.
```

Comment:
"Please describe the bug or your proposed change to the software `in as much detail` as possible. Please provide screenshots when possible."